Makefile

```diff
@@ -21,7 +21,8 @@ test-assignment:
 	@cp $(EXERCISESDIR)/$(ASSIGNMENT)/$(EXAMPLE) $(OUTDIR)/$(TESTFILE)
 	@sed 's/pending/it/g' $(EXERCISESDIR)/$(ASSIGNMENT)/$(SPECFILE) > $(OUTDIR)/$(SPECFILE)
 	@cd $(OUTDIR) && crystal spec $(SPECFILE)
-	@echo "\n"
+	crystal tool format --check $(EXERCISESDIR)/$(ASSIGNMENT)/$(SPECFILE) $(EXERCISESDIR)/$(ASSIGNMENT)/$(EXAMPLE)
```

Comment:
Yes, I flipped between the two. I guess the question is whether there'll be any files it would pick up that you don't want to be reformatted, like any templates or I don't know. You know this better than me though, so if you think it should just pickup everything I'll change it.